docs/guides.md

```diff
@@ -50,6 +50,10 @@ The virtual display is accelerated by the NVidia GPU using the TwinView configur
 I also only tested this on an Artix runit init system on LAN.
 I didn't have to do anything special with pulseaudio (pipewire untested).
 
+Edit by MidwesternRodent on 10/10/2024: Pipewire does not seem to work when sunshine is started over an SSH session.
```

Comment:
Personally, I don't think my modifications to that guide warrant attribution on the document header so if it's all the same I'd prefer it to just be left to Eric Dong as they did all the work on that. I agree it's much cleaner without the edit notice though, and in retrospect I don't think a modification that minor (or the visudo changes) is really putting my words in their mouth.

@e-dong - Sorry for the ping but just wanted to give you heads up on some incredibly minor changes I'm suggesting to your guide for remote SSH setup of sunshine and give you an opportunity to provide some input if you'd like. Since your name's on the guide, I just want to make sure I don't speak for you.

Just calling out some problems with pipewire audio when sunshine is started over SSH, and directing people to the guide this PR is about if they want to address them. I also made some tweaks to the sudoers.d session to direct people to use visudo rather than echoing to the sudoers file, but that's a semantics / best practice disagreement I'd be happy to drop if you'd prefer to keep the original wording.

Reply:
Sounds fine to me.

Also, as a heads up... I'm hoping to create a blog (powered by jekyll and gh-pages) soonish. Once that's ready these guides will probably be migrated there. I don't have timing on that though, so this should continue here for now.